Gemfile.lock

```diff
@@ -64,7 +64,9 @@ GEM
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
-    bootstrap_form (2.7.0)
+    bootstrap_form (4.2.0)
```

Comment:
oh wow, I'm a little surprised this works with our version of bootstrap. I assume this is probably the thing that's causing the very slight view changes, yeah?

Reply:
Yep!